apps/web-giddh/src/app/expenses/components/pending-list/pending-list.component.html

```diff
@@ -66,7 +65,8 @@
                             {{expenses.particularAccount.name}}
                         </td>
 
-                        <td [attr.data-title]="commonLocaleData?.app_amount" class="text-right pr-2"> {{expenses.currencySymbol}}
+                        <td [attr.data-title]="commonLocaleData?.app_amount" class="text-right pr-2">
+                            {{expenses.currencySymbol}}
                             {{ expenses.amount | giddhCurrency}}</td>
```

Comment:
implement amount-field component here

Reply:
Not able to found currency code on JSON ,